Merge and energy consumption pages pre-merge updates (#6599)
ethereum/ethereum-org-website: src/content/energy-consumption/index.md

Since inception, Ethereum has aimed to implement a PoS consensus mechanism, but doing this without sacrificing security and decentralization has taken years of focused research and development. Therefore, the network started by using PoW consensus. PoW consensus requires miners to use their computing hardware to solve a puzzle. The solution to this puzzle acts as proof that some energy has been expended by the miner which in turn demonstrates that they have invested some real-world value for the right to add to the blockchain. PoW is just a mechanism to decide who gets to add the next block, as is PoS. Swapping PoW for PoS, where the real world value invested comes from ether staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.

## The Merge {#the-merge}

> A PoS blockchain known as the "Beacon Chain" has been running since November 2020 alongside the PoW Ethereum Mainnet. In 2022 several custom devnets have been merged with their own Beacon Chains to aid in debugging Ethereum clients in advance of the real merge. More recently, Ethereum developers have started merging public testnets (Kiln, Kintsugi, Ropsten) and shadow-forks (forks of Ethereum Mainnet) as the final stage of testing. Merging Ethereum Mainnet with the Beacon Chain is expected to happen in the second half of 2022. At the moment of the merge, Ethereum's PoW mining will be switched off, PoS consensus will take over, and the energy consumed by the network will drop to <0.05% of its pre-merge amount.

Reviewer: Is it worth mentioning Goerli and Sepolia in some capacity here?

Reviewer 2: I would maybe emphasize the PoS chain only ran the consensus algorithm, and didn't run user transactions. "PoS blockchain" sort of implies another version of Ethereum was launched. 

> In 2022 several custom devnets have been merged with their own Beacon Chains to aid in debugging Ethereum clients in advance of the real merge.

I would rephrase this to: 

> In 2022, several proof-of-work devnets were launched and transitioned to proof-of-stake by merging with their own Beacon Chains. These helped client teams test software before moving to longer-lived networks. 

> More recently, Ethereum developers have started merging public testnets (Kiln, Kintsugi, Ropsten) and shadow-forks (forks of Ethereum Mainnet) as the final stage of testing.

Similarly, I'd suggest:

> After these devnets, Ethereum developers launched new testnets for the community to use (Kiln & Kintsugi) and ran multiple [shadow forks](https://etherworld.co/2022/04/20/ethereum-mainnet-shadow-forking-an-overview/) of existing testnets and mainnet. Now that these have stabilized, teams are moving to the final stages of testing: transitioning long-lived testnets (Ropsten, Goerli, Sepolia) to proof-of-stake.

Reviewer 3: thanks, updated accordingly

The ability for any miner to add new blocks only works if there is a cost associated with mining and unpredictability about which specific node submits the next block. These conditions are met by imposing PoW. To be eligible to submit a block of transactions, a miner must be the first to submit the solution to a computationally expensive puzzle. To successfully take control of the blockchain, a dishonest miner would have to consistently win the PoW race by investing in sufficient hardware and energy to outperform the majority of other miners.

This mechanism of securing the network is problematic for several reasons. Miners can increase their odds of success by investing in more and more powerful hardware, creatign conditions for an arms race with miners acquiring increasingly power-hungry mining equipment. This increases the network's energy consumption and generates hardware waste. Ethereum's PoW protocol currently has a total annualized power consumption approximately equal to that of Finland <sup>[^1]</sup> and carbon footprint similar to Switzerland<sup>[^1]</sup>.

> PoS uses validators instead of miners. Validators perform the same function as miners except that instead of expending their assets up-front in the form of energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ether cna be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the PoW network comes from the mining algorithm, switching to PoS reduces the energy expenditure dramatically. There is also no benefit to be had by investing in mroe powerful hardware under PoS, so there is no arms-race condition and less electronic waste. Ethereum validators can run on normal laptops or even low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).

Reviewer: ```suggestion
PoS uses validators instead of miners. Validators perform the same function as miners except that instead of expending their assets up-front in the form of energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ether can be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the PoW network comes from the mining algorithm, switching to PoS reduces the energy expenditure dramatically. There is also no benefit to be had by investing in mroe powerful hardware under PoS, so there is no arms-race condition and less electronic waste. Ethereum validators can run on normal laptops or even low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).
```

> PoS uses validators instead of miners. Validators perform the same function as miners except that instead of expending their assets up-front in the form of energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ether cna be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the PoW network comes from the mining algorithm, switching to PoS reduces the energy expenditure dramatically. There is also no benefit to be had by investing in mroe powerful hardware under PoS, so there is no arms-race condition and less electronic waste. Ethereum validators can run on normal laptops or even low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).

Reviewer 4: My 2 gwei: if we're talking about slashing on the energy consumption page, we've gone too deep 😅

> PoS uses validators instead of miners. Validators perform the same function as miners except that instead of expending their assets up-front in the form of energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ether can be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the PoW network comes from the mining algorithm, switching to PoS reduces the energy expenditure dramatically. There is also no benefit to be had by investing in more powerful hardware under PoS, so there is no arms-race condition and less electronic waste. Ethereum validators can run on normal laptops or even low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).

Author: https://ethereum.org/en/contributing/style-guide/#acronyms
Mentioned this before, we introduce these terms at the top: "proof-of-work (PoW) to proof-of-stake (PoS)"
Can we not use these acronyms throughout the page? Personally feel it improves readability.

Read more detailed information about [how PoS is implemented in Ethereum](/developers/docs/consensus-mechanisms/pos) and how it compares to PoW.

## Proof-of-stake energy expenditure {#proof-of-stake-energy}

Estimates based on the current Beacon Chain suggest that the merge to PoS could result in a 99.95% reduction in total energy use, with PoS being ~2000x more energy-efficient than PoW. The energy expenditure of Ethereum will be roughly equal to the cost of running a modest laptop for each node on the network.

Many articles estimate "per-transaction" energy expenditure to compare blockchains to other industries. The benefit of this is that it is easy to understand, but the energy required to mine a block is independent of the number of transactions within it. A per transaction unit of energy expenditure implies that fewer transactions would lead to smaller energy expenditure, and vice-versa, which is not the case. A per-transaction estimate is highly dependent upon how a blockchain's transaction throughput is defined, and tweaking this definition can be gamed to make the value seem larger or smaller.

> For example, for Ethereum the transaction throughput is not only that of the base layer - it is also the sum of the transaction throughput of all of its “layer 2” rollups, which are not generally included in calculations and would drastically reduce them. This is why tools that compare energy consumption per transaction across platforms are misleading.

Reviewer: ```suggestion
For example, for Ethereum the transaction throughput is not only that of the base layer - it is also the sum of the transaction throughput of all of its “[layer 2](/layer-2/)” rollups, which are not generally included in calculations and would drastically reduce them. This is why tools that compare energy consumption per transaction across platforms are misleading.
```

Might be a good place to link out to L2 page

Whole-network energy consumption and carbon footprints for Bitcoin and Ethereum are estimated at [Digiconomist](https://digiconomist.net/ethereum-energy-consumption). At the time of writing this article Ethereum’s total energy consumption is ~112 TWh/yr, equivalent to that of the Netherlands, with a Carbon emission equivalent to that of Singapore (53 MT/yr). For comparison, Bitcoin currently expends about 200 TWh/yr energy and emits about 100 MT/yr C, while also generating about 32,000 T of electrical waste from obsolete hardware per year. Switching off Ethereum’s PoW in favor of PoS will reduce this energy expenditure by more than 99.95%, implying that the total energy expenditure for securing Ethereum is expected to be closer to **0.01 TWh/yr**.

> To put this in context we can compare to annualized estimates for other industries. If we take Ethereum to be a platform for securely holding digital assets as investments, perhaps we can compare to mining gold, which has been estimated to expend about [240 TWh/yr](https://www.kitco.com/news/2021-05-17/Gold-s-energy-consumption-doubles-that-of-bitcoin-Galaxy-Digital.html). As a digital payments platform we could perhaps compare to PayPal (about [0.26 TWh/yr](https://app.impaakt.com/analyses/paypal-consumed-264100-mwh-of-energy-in-2020-24-from-non-renewable-sources-27261)). As an entertainment platform we could perhaps compare to the gaming industry which has been estimated to expend about [34 TW/yr](https://www.researchgate.net/publication/336909520_Toward_Greener_Gaming_Estimating_National_Energy_Use_and_Energy_Efficiency_Potential), or Netflix which expends about [94 TWh/yr](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix), or Youtube which has been estimated to expend about [244 TWh/yr](https://thefactsource.com/how-much-electricity-does-youtube-use/). Estimates of Youtube’s energy expenditure have even been broken down by channel and even further by individual videos. [Those estimates](https://thefactsource.com/how-much-electricity-does-youtube-use/) imply that at least 45x more energy was expended watching Gangnam Style in 2019 than will be expended in a year of securing PoS Ethereum.

Reviewer 5: ```suggestion
To put Ethereum's energy consumption in context, we can compare annualized estimates for other industries. If we take Ethereum to be a platform for securely holding digital assets as investments, perhaps we can compare to mining gold, which has been estimated to expend about [240 TWh/yr](https://www.kitco.com/news/2021-05-17/Gold-s-energy-consumption-doubles-that-of-bitcoin-Galaxy-Digital.html). As a digital payments platform we could perhaps compare to PayPal (about [0.26 TWh/yr](https://app.impaakt.com/analyses/paypal-consumed-264100-mwh-of-energy-in-2020-24-from-non-renewable-sources-27261)). As an entertainment platform we could perhaps compare to the gaming industry which has been estimated to expend about [34 TW/yr](https://www.researchgate.net/publication/336909520_Toward_Greener_Gaming_Estimating_National_Energy_Use_and_Energy_Efficiency_Potential), or Netflix which expends about [94 TWh/yr](https://www.carbonbrief.org/factcheck-what-is-the-carbon-footprint-of-streaming-video-on-netflix), or Youtube which has been estimated to expend about [244 TWh/yr](https://thefactsource.com/how-much-electricity-does-youtube-use/). Estimates of Youtube's energy expenditure have been broken down by channel and individual videos. [Those estimates](https://thefactsource.com/how-much-electricity-does-youtube-use/) imply that people consumed 45 times more energy watching Gangnam Style in 2019 than proof-of-stake Ethereum will use in a year.
```

## A greener Ethereum {#green-ethereum}

> While Ethereum's energy consumption has historically been substantial, there has been a major investment of developer time and intellect into transitioning from energy-hungry to energy-efficient block production. To quote [Bankless](http://podcast.banklesshq.com/), the best way to reduce the energy consumed by PoW is simply to "turn it off", which is the approach Ethereum has committed to take. At the same time, there is a substantial, growing and highly active "ReFi" (Regenerative Finance) community building on Ethereum. ReFi refers to applications that use DeFi components to build financial applicatiosn that have positive externalities benfiting the environment. This is part of a wider "Solarpunk" movement that is closely aligned with Ethereum and aims to couple technological advancement and environmental stewardship. The decentralized, permissionless, composable nature of Ethereum makes it the ideal base layer for the ReFi and solarpunk communities. After the merge, the technology and philosophy of Ethereum will finally reconcile and Ethereum should become an overall environmentally-positive technology.

Reviewer 5: ```suggestion
While Ethereum's energy consumption has historically been substantial, there has been a significant investment of developer time and intellect into transitioning from energy-hungry to energy-efficient block production. To quote [Bankless](http://podcast.banklesshq.com/), the best way to reduce the energy consumed by proof-of-work is simply to "turn it off", which is the approach Ethereum has committed to taking. 

At the same time, there is a substantial, growing, and highly active **regenerative finance (ReFI)** community building on Ethereum. ReFi applications use DeFi components to build financial applications that have positive externalities benefiting the environment. ReFi is part of a wider "Solarpunk" movement that is closely aligned with Ethereum and aims to couple technological advancement and environmental stewardship. The decentralized, permissionless, composable nature of Ethereum makes it the ideal base layer for the ReFi and solarpunk communities. After The Merge, the technology and philosophy of Ethereum will finally reconcile, and Ethereum should become an environmentally-positive technology.
```

# Ethereum energy consumption {#introduction}

> Ethereum's energy consumption will be reduced by [~99.95%](https://blog.ethereum.org/2021/05/18/country-power-no-more/) following [The Merge](/upgrades/merge/index.md) from [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/#proof-of-work) to [proof-of-stake (PoS)](/developers/docs/consensus-mechanisms/pos). After The Merge Ethereum will use dramatically less carbon to be more secure.

Reviewer 4: ```suggestion
Ethereum's energy consumption will be reduced by [~99.95%](https://blog.ethereum.org/2021/05/18/country-power-no-more/) following [The Merge](/upgrades/merge/index.md) from [proof-of-work (PoW)](/developers/docs/consensus-mechanisms/#proof-of-work) to [proof-of-stake (PoS)](/developers/docs/consensus-mechanisms/pos). After The Merge, Ethereum will use dramatically less carbon to be more secure.
```

> Since its inception, Ethereum has aimed to implement a proof-of-stake consensus mechanism, but doing this without sacrificing security and decentralization has taken years of focused research and development. Therefore, the network started by using proof-of-work consensus. Proof-of-work consensus requires miners to use their computing hardware to solve a puzzle. The solution to the puzzle proves that some energy has been expended by the miner, demonstrating that they have invested real-world value for the right to add to the blockchain. Proof-of-work is just a mechanism to decide who gets to add the next block, as is proof-of-stake. Swapping proof-of-work for proof-of-stake, where the real world value invested comes from ETH staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.

Reviewer 5: ```suggestion
Since its inception, Ethereum has aimed to implement a proof-of-stake consensus mechanism, but doing this without sacrificing security and decentralization has taken years of focused research and development. Therefore, the network started by using proof-of-work consensus. Proof-of-work consensus requires miners to use their computing hardware to solve a puzzle. The solution to the puzzle proves that energy has been expended by the miner, demonstrating that they invested real-world value for the right to add to the blockchain. Both proof-of-work and proof-of-stake are just mechanisms to decide who gets to add the next block. Swapping proof-of-work for proof-of-stake, where the real-world value invested comes from ETH staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.
```

## The Merge {#the-merge}

> A blockchain known as the "Beacon Chain" has been running the consensus algorithm since November 2020 alongside the PoW Ethereum Mainnet. In 2022, several proof-of-work devnets were launched and transitioned to proof-of-stake by merging with their own Beacon Chains. These helped client teams test software before moving to longer-lived networks. After these devnets, Ethereum developers launched new testnets for the community to use (Kiln & Kintsugi) and ran multiple shadow forks of existing testnets and mainnet. Now that these have stabilized, teams are moving to the final stages of testing: transitioning long-lived testnets (Ropsten, Goerli, Sepolia) to proof-of-stake. Merging Ethereum Mainnet with the Beacon Chain is expected to happen in the second half of 2022. At the moment of the merge, Ethereum's PoW mining will be switched off, PoS consensus will take over, and the energy consumed by the network will drop to <0.05% of its pre-merge amount.

Reviewer 5: > transitioned to proof-of-stake by merging with their own Beacon Chains.

This isn't easy to understand

> A blockchain known as the "Beacon Chain" has been running the consensus algorithm since November 2020 alongside the PoW Ethereum Mainnet. In 2022, several proof-of-work devnets were launched and transitioned to proof-of-stake by merging with their own Beacon Chains. These helped client teams test software before moving to longer-lived networks. After these devnets, Ethereum developers launched new testnets for the community to use (Kiln & Kintsugi) and ran multiple shadow forks of existing testnets and mainnet. Now that these have stabilized, teams are moving to the final stages of testing: transitioning long-lived testnets (Ropsten, Goerli, Sepolia) to proof-of-stake. Merging Ethereum Mainnet with the Beacon Chain is expected to happen in the second half of 2022. At the moment of the merge, Ethereum's PoW mining will be switched off, PoS consensus will take over, and the energy consumed by the network will drop to <0.05% of its pre-merge amount.

Reviewer 5: ```suggestion
[The Beacon Chain](/upgrades/beacon-chain/) has been running the proof-of-stake since November 2020 alongside the proof-of-work Ethereum Mainnet. In 2022, Ethereum developers transitioned several **testing networks (testnets)** running proof-of-stake to proof-of-stake by merging with their own Beacon Chains. These helped client teams test the software before moving to longer-lived networks. After these testnets, Ethereum developers launched new testnets for the community to use (Kiln & Kintsugi) and ran multiple shadow forks of existing testnets and Mainnet. Now that these have stabilized, teams are moving to the final stages of testing: transitioning long-lived testnets (Ropsten, Goerli, Sepolia) to proof-of-stake. Merging Ethereum Mainnet with the Beacon Chain is expected to happen in the second half of 2022. At the moment of the merge, Ethereum's proof-of-work mining will be switched off, proof-of-stake consensus will take over, and the energy consumed by the network will drop to <0.05% of its pre-merge amount.
```

## Why proof-of-stake is greener than proof-of-work {#why-pos-is-greener-than-pow}

> PoW is a robust way to secure the network. Transactions on the Ethereum blockchain are validated by [miners](/developers/docs/consensus-mechanisms/pow/mining). Miners bundle together transactions into ordered blocks and add them to the Ethereum blockchain. The new blocks get broadcast to all the other node operators who run the transactions independently and verify that they are valid. Any dishonesty shows up as an inconsistency between different nodes. Honest blocks are added to the blockchain and become an immutable part of history.

Reviewer 5: ```suggestion
Proof-of-work is a robust way to secure the network. Transactions on the Ethereum blockchain are validated by [miners](/developers/docs/consensus-mechanisms/pow/mining). Miners bundle together transactions into ordered blocks and add them to the Ethereum blockchain. The new blocks get broadcast to all the other node operators who run the transactions independently and verify that they are valid. Any dishonesty shows up as an inconsistency between different nodes. Honest blocks are added to the blockchain and become an immutable part of history.
```

> The ability for any miner to add new blocks only works if there is a cost associated with mining and unpredictability about which specific node submits the next block. These conditions are met by imposing PoW. To be eligible to submit a block of transactions, a miner must be the first to submit the solution to a computationally expensive puzzle. To successfully take control of the blockchain, a dishonest miner would have to consistently win the PoW race by investing in sufficient hardware and energy to outperform the majority of other miners.

Reviewer 5: ```suggestion
The ability for any miner to add new blocks only works if there is a cost associated with mining and unpredictability about which specific node submits the next block. These conditions are met by imposing proof-of-work. To be eligible to submit a block of transactions, a miner must be the first to submit the solution to a computationally expensive puzzle. To successfully take control of the blockchain, a dishonest miner would have to consistently win the proof-of-work race by investing in sufficient hardware and energy to outperform the majority of other miners.
```

> This mechanism of securing the network is problematic for several reasons. Miners can increase their odds of success by investing in more and more powerful hardware, creating conditions for an arms race with miners acquiring increasingly power-hungry mining equipment. This increases the network's energy consumption and generates hardware waste. Ethereum's PoW protocol currently has a total annualized power consumption approximately equal to that of Finland <sup>[^1]</sup> and carbon footprint similar to Switzerland<sup>[^1]</sup>.

Reviewer 5: ```suggestion
This mechanism of securing the network is problematic for several reasons. First, miners can increase their odds of success by investing in more powerful hardware, creating conditions for an arms race with miners acquiring increasingly power-hungry mining equipment. This increases the network's energy consumption and generates hardware waste. Second, Ethereum's proof-of-work protocol currently has a total annualized power consumption approximately equal to that of Finland <sup>[^1]</sup> and a carbon footprint similar to Switzerland<sup>[^1]</sup>.
```

> PoS uses validators instead of miners. Validators perform the same function as miners except that instead of expending their assets up-front in the form of energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ether can be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the PoW network comes from the mining algorithm, switching to PoS reduces the energy expenditure dramatically. There is also no benefit to be had by investing in more powerful hardware under PoS, so there is no arms-race condition and less electronic waste. Ethereum validators can run on normal laptops or even low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).

Reviewer 5: ```suggestion
Proof-of-stake uses validators instead of miners. Validators perform the same function as miners, except that instead of expending their assets up-front as energy expenditure, they stake ETH as collateral against dishonest behavior. This staked ETH can be destroyed if the validator misbehaves, with more severe penalties for more nefarious actions. This strongly incentivizes active and honest participation in securing the network without requiring large energy expenditure. Since almost all of the energy expended securing the proof-of-work network comes from the mining algorithm, switching to proof-of-stake dramatically reduces energy expenditure. There is also no benefit to be had by investing in more powerful hardware under proof-of-stake, so there is no arms-race condition and less electronic waste. Ethereum validators can run on typical laptops or low-power devices such as [Raspberry Pi](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/user-guide/ethereum2.0.html).
```

> Read more detailed information about [how PoS is implemented in Ethereum](/developers/docs/consensus-mechanisms/pos) and how it compares to PoW.

Reviewer 5: ```suggestion
Read more on [how Ethereum implements proof-of-stake](/developers/docs/consensus-mechanisms/pos) and how it compares to proof-of-work.
```

## Proof-of-stake energy expenditure {#proof-of-stake-energy}

> Estimates based on the current Beacon Chain suggest that the merge to PoS could result in a 99.95% reduction in total energy use, with PoS being ~2000x more energy-efficient than PoW. The energy expenditure of Ethereum will be roughly equal to the cost of running a modest laptop for each node on the network.

Reviewer 5: ```suggestion
Estimates based on the current Beacon Chain suggest that The Merge to proof-of-stake could result in a 99.95% reduction in total energy use, with proof-of-stake being ~2000x more energy-efficient than proof-of-work. The energy expenditure of Ethereum will be roughly equal to the cost of running a modest laptop for each node on the network.
```

> Many articles estimate "per-transaction" energy expenditure to compare blockchains to other industries. The benefit of this is that it is easy to understand, but the energy required to mine a block is independent of the number of transactions within it. A per transaction unit of energy expenditure implies that fewer transactions would lead to smaller energy expenditure, and vice-versa, which is not the case. A per-transaction estimate is highly dependent upon how a blockchain's transaction throughput is defined, and tweaking this definition can be gamed to make the value seem larger or smaller.

Reviewer 5: ```suggestion
Many articles estimate "per-transaction" energy expenditure to compare blockchains to other industries. The benefit of this is that it is easy to understand, but the energy required to mine a block is independent of the number of transactions within it. A per transaction unit of energy expenditure implies that fewer transactions would lead to smaller energy expenditure and vice-versa, which is not the case. A per-transaction estimate is highly dependent upon how a blockchain's transaction throughput is defined, and tweaking this definition can be gamed to make the value seem larger or smaller.
```

> For example, for Ethereum the transaction throughput is not only that of the base layer - it is also the sum of the transaction throughput of all of its “[layer 2](/layer-2/)” rollups, which are not generally included in calculations and would drastically reduce them. This is why tools that compare energy consumption per transaction across platforms are misleading.

Reviewer 5: ```suggestion
For example, on Ethereum, the transaction throughput is not only that of the base layer - it is also the sum of the transaction throughput of all of its "[layer 2](/layer-2/)" rollups, which are not generally included in calculations and would drastically reduce them. This is why tools that compare energy consumption per transaction across platforms are misleading.
```

> More relevant is the overall energy consumption and carbon footprint of the network as a whole. From those values one can examine what that network offers to its users and to society at large and make a more holistic evaluation of whether that energy expenditure is justified or not. Per transaction measurements, on the other hand, imply the value of the network only comes from its role in transferring crypto between accounts and prohibit an honest cost-benefit analysis.

Reviewer 5: ```suggestion
More relevant is the overall energy consumption and carbon footprint of the network as a whole. From those values, we can examine what that network offers to its users and society at large and make a more holistic evaluation of whether that energy expenditure is justified or not. Per transaction measurements, on the other hand, imply the value of the network only comes from its role in transferring crypto between accounts and prohibits an honest cost-benefit analysis.
```

> Whole-network energy consumption and carbon footprints for Bitcoin and Ethereum are estimated at [Digiconomist](https://digiconomist.net/ethereum-energy-consumption). At the time of writing this article Ethereum’s total energy consumption is ~112 TWh/yr, equivalent to that of the Netherlands, with a Carbon emission equivalent to that of Singapore (53 MT/yr). For comparison, Bitcoin currently expends about 200 TWh/yr energy and emits about 100 MT/yr C, while also generating about 32,000 T of electrical waste from obsolete hardware per year. Switching off Ethereum’s PoW in favor of PoS will reduce this energy expenditure by more than 99.95%, implying that the total energy expenditure for securing Ethereum is expected to be closer to **0.01 TWh/yr**.

Reviewer 5: ```suggestion
[Digiconomist provides whole-network energy consumption and carbon footprints for Bitcoin and Ethereum](https://digiconomist.net/ethereum-energy-consumption). At the time of writing this article, Ethereum's total energy consumption is ~112 TWh/yr, comparable to that of the Netherlands, with a Carbon emission equivalent to that of Singapore (53 MT/yr). For comparison, Bitcoin currently expends about 200 TWh/yr of energy and emits about 100 MT/yr C, while generating about 32,000 T of electrical waste from obsolete hardware annually. Switching off Ethereum's proof-of-work in favor of proof-of-stake will reduce this energy expenditure by more than 99.95%, implying that the total energy expenditure for securing Ethereum is closer to **0.01 TWh/yr**.
```
